app/views/laa_references/_form.html.haml

```diff
@@ -17,7 +17,7 @@
     label: { text: t('laa_reference.link.form.maat_reference.label') },
     hint: { text: t('laa_reference.link.form.maat_reference.hint') }
 
-  = f.govuk_submit(t('laa_reference.link.form.submit'))
+  = f.govuk_submit t('laa_reference.link.form.submit'), prevent_double_click: true, data: { disable_with: 'Linking...' }
```

Comment:
Just checking;

* Is `prevent-double-click` required in addition to the `data-disable-with`?
* Isn't `true` the default value for `prevent-double-click`?